src/views/network/view_session.cpp

```diff
@@ -44,6 +45,27 @@ namespace big
 		ImGui::Checkbox("Disable Filter", &g->session.disable_chat_filter);
 		ImGui::Checkbox("Log Chat Messages", &g->session.log_chat_messages);
 		ImGui::Checkbox("Log Text Messages", &g->session.log_text_messages);
+		static char msg[256];
+		ImGui::InputText("##message", msg, sizeof(msg));
+		ImGui::SameLine();
+		ImGui::Checkbox("Is Team", &g->session.is_team);
+		ImGui::SameLine();
+		components::button("Send", []
+		{
+            if(const auto net_game_player = gta_util::get_network_player_mgr()->m_local_net_player; net_game_player)
+			{
+                if(g_pointers->m_send_chat_message(*g_pointers->m_send_chat_ptr, net_game_player->get_net_data(), msg, g->session.is_team))
+				{
+					int scaleform = GRAPHICS::REQUEST_SCALEFORM_MOVIE("MULTIPLAYER_CHAT");
+					GRAPHICS::BEGIN_SCALEFORM_MOVIE_METHOD(scaleform, "ADD_MESSAGE");
+					GRAPHICS::SCALEFORM_MOVIE_METHOD_ADD_PARAM_PLAYER_NAME_STRING(net_game_player->get_name()); // player name
+					GRAPHICS::SCALEFORM_MOVIE_METHOD_ADD_PARAM_LITERAL_STRING(msg); // content
+					GRAPHICS::SCALEFORM_MOVIE_METHOD_ADD_PARAM_TEXTURE_NAME_STRING(HUD::GET_FILENAME_FOR_AUDIO_CONVERSATION(g->session.is_team ? "MP_CHAT_TEAM" : "MP_CHAT_ALL")); // scope
+					GRAPHICS::DRAW_SCALEFORM_MOVIE_FULLSCREEN(scaleform, 255, 255, 255, 255, 0);
+					GRAPHICS::END_SCALEFORM_MOVIE_METHOD();
```

Comment:
Make this a function under `util/`, maybe in session.hpp.

Reply:
Or `notify.hpp`.

Reply:
Yeah, that's also an option.